setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = C901, W503, E203, F841, E712
```

Comment:
Too much ignoring, why not address (at least some) of them?

Reply:
Had a look, two down, would keep ignoring the rest.